tests/pass-dep/libc/libc-eventfd.rs

```diff
@@ -1,4 +1,5 @@
 //@only-target: linux
+//@only-target: android
```

Comment:
You didn't enable the test on Android so this change does nothing.

Reply:
Added an issue: https://github.com/rust-lang/miri/issues/3996.